### PR TITLE
Rework counter into refcounter

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+        this.counters = counters;
+        this.populateWikimedias();
+        this.populateReferenceProperties();
+    }
+
+    /**
+     * The below list was generated using a SPARQL query.
+     * FIXME: there is probably a better way to store, manage and update this....
+     */
+    private void populateWikimedias() {
```
Comment (JeroenDeDauw) [inline java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java]: :D

Comment (addshore) [inline java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java]: Yeh! :)
Also, woo tests!
I'll have to spend some time working out how to make travis run the java tests with maven at some point (unless you want to) ;)

Comment (JeroenDeDauw) [inline java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java]: Not that hard... https://github.com/Wikidata/Wikidata-Toolkit/blob/master/.travis.yml

Comment (addshore) [inline java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java]: Oh wait... I have already set them up....... https://travis-ci.org/wmde/wikidata-analysis/builds/96973104 heh

Comment (JeroenDeDauw) [inline java/analyzer/src/main/java/org/wikidata/analyzer/Processor/ReferenceProcessor.java]: Perhaps that was the other addshore?

**Pull request merged**